README.md

```diff
@@ -35,3 +35,37 @@ If using without Pants, run `flake8 your_module.py` [as usual](http://flake8.pyc
 ## Migration from `pantsbuild.pants.contrib.python.checks.checker`
 
 TODO
+
+## Developers Guide
+
+We use [Poetry](https://python-poetry.org) for dependency management, building the wheel, and publishing.
+
+### To install
+
+```bash
+$ poetry install
+```
+
+This will use whichever version `python` points to. You can set `poetry env use python3.7`, for example, to point to a different Python discoverable on your PATH.
+
+### To test
+
+```bash
+$ poetry run pytest
+```
+
+You may also manually test by building a PEX with this plugin, as follows:
+
+```bash
+$ poetry build
+$ pex flake8 dist/flake8_pantsbuild-0.1.0-py2.py3-none-any.whl --entry-point flake8 --interpreter-constraint='CPython>=3.6' --output-file flake8.pex
```

Comment:
`pex flake8 . --entry-point ...` should be enough, pex now knows PEP-517/PEP-518.

Reply:
TIL. Thanks!